src/fu-util-common.c

```diff
@@ -65,10 +65,14 @@ fu_util_get_expected_command(const gchar *target)
 gboolean
 fu_util_using_correct_daemon(GError **error)
 {
-#ifdef HAVE_SYSTEMD
 	g_autofree gchar *default_target = NULL;
 	g_autoptr(GError) error_local = NULL;
-	const gchar *target = fu_util_get_systemd_unit();
+	const gchar *target;
```

Comment:
I think you've got to be a bit more careful here e.g. error_local, default_target are unused in the non HAVE_SYSTEMD case.

Reply:
fixed.  I forgot we just return TRUE in the non-systemd case